RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -16,7 +15,7 @@ public class AddTestModuleCommand : CommandBase
         private readonly VBE _vbe;
         private readonly NewUnitTestModuleCommand _command;
 
-        public AddTestModuleCommand(VBE vbe, RubberduckParserState state, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
+        public AddTestModuleCommand(VBE vbe, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
```

Comment:
The second is the one that actually adds either test to the module.  The first (and it's expected error equivalent) both just call the other.